### PR TITLE
Add ntp

### PR DESCRIPTION
**Pull request by BMikh0**

### CODE REVIEW

File: projects/ntp/ntpd_receive.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+################################################################################
+
+[sydr]
+args = "-l debug -j2"
+target = "/ntpd_receive_sydr @@"
+jobs = 2
+
+[libfuzzer]
+path = "/ntpd_receive_libfuzzer"
+args = "-rss_limit_mb=0 /corpus"
```
Comment (SweetVishnya) [inline projects/ntp/ntpd_receive.toml:24]: why do we need rss_limit_mb=0 ?

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

**Comment by PaDarochek** [comment]
Make build.sh executable  
`chmod +x build.sh`

File: projects/ntp/ntpd_receive-afl++.toml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#
+################################################################################
+
+[sydr]
+args = "-l debug -j2"
+target = "/ntpd_receive_sydr @@"
+jobs = 2
+
+[aflplusplus]
+target = "ntpd_receive_aflplusplus"
```
Comment (PaDarochek) [inline projects/ntp/ntpd_receive-afl++.toml:23]: Add / before the filename in target

File: projects/ntp/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+RUN chmod +x bk-7.3.3-x86_64-glibc213-linux.bin
+RUN ./bk-7.3.3-x86_64-glibc213-linux.bin /usr/local/bitkeeper
+RUN ln -s /usr/local/bitkeeper/bk /usr/local/bin/bk
+
+WORKDIR /src
+
+RUN bk clone -r1.3714 http://bk.ntp.org/ntp-dev
+
+COPY build.sh patch.diff /src
```
Comment (SweetVishnya) [inline projects/ntp/Dockerfile:32]: Step 9/12 : COPY build.sh patch.diff /src
When using COPY with more than one source file, the destination must be a directory and end with a /

File: projects/ntp/ntpd_receive-afl++.toml (hunk 2 above)
Comment (SweetVishnya) [reply projects/ntp/ntpd_receive-afl++.toml:23]: Add @@:
```
/ntpd_receive_aflplusplus @@
```

Comment (PaDarochek) [inline projects/ntp/ntpd_receive-afl++.toml:23]: Add @@ after target name

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**